### PR TITLE
feat: Add sophisticated defensive aave seed

### PR DESCRIPTION
**Pull request by miguelmtzinf**

### CODE REVIEW

File: src/20240528_Multi_GHOCrossChainLaunch/AaveV3Arbitrum_GHOCrossChainLaunch_20240528.sol [hunk 1]
```diff
@@ -225,12 +223,68 @@ contract AaveV3Arbitrum_GHOCrossChainLaunch_20240528 is AaveV3PayloadArbitrum {
...
   function _defensiveSeed() internal {
-    // Add Governance as Facilitator
-    IGhoToken(GHO).addFacilitator(address(this), 'Governance', uint128(Utils.GHO_SEED_AMOUNT));
+    AaveDefensiveSeed defensiveSeed = new AaveDefensiveSeed(GHO, Utils.GHO_SEED_AMOUNT);
+
+    // Add Facilitator and remove just after the mint of seed amount
+    uint256 seedAmount = defensiveSeed.DEFENSIVE_SEED_AMOUNT();
+    IGhoToken(GHO).addFacilitator(address(defensiveSeed), 'DefensiveSeed', uint128(seedAmount));
+    defensiveSeed.mint();
+    IGhoToken(GHO).setFacilitatorBucketCapacity(address(defensiveSeed), 0);
```
Comment (brotherlymite) [inline src/20240528_Multi_GHOCrossChainLaunch/AaveV3Arbitrum_GHOCrossChainLaunch_20240528.sol:232]:
```suggestion
```
@miguelmtzinf This is not needed no?

File: src/20240528_Multi_GHOCrossChainLaunch/AaveV3Arbitrum_GHOCrossChainLaunch_20240528.sol [hunk 2]
```diff
@@ -225,12 +223,68 @@ contract AaveV3Arbitrum_GHOCrossChainLaunch_20240528 is AaveV3PayloadArbitrum {
...
+
+    // Give FacilitatorManager role so it can unwind
+    IGhoToken(GHO).grantRole(IGhoToken(GHO).FACILITATOR_MANAGER_ROLE(), address(defensiveSeed));
+  }
+}
+
+contract AaveDefensiveSeed {
```
Comment (brotherlymite) [inline src/20240528_Multi_GHOCrossChainLaunch/AaveV3Arbitrum_GHOCrossChainLaunch_20240528.sol:239]: @miguelmtzinf I would document the contract a bit, mentioning `mint()` method being called by the payload and the `burn()` method which is permissionless and should be called after address(0) has sufficient aGHO balance.

File: src/20240528_Multi_GHOCrossChainLaunch/AaveV3Arbitrum_GHOCrossChainLaunch_20240528.sol (hunk 1 above)
Comment (miguelmtzinf) [reply src/20240528_Multi_GHOCrossChainLaunch/AaveV3Arbitrum_GHOCrossChainLaunch_20240528.sol:232]: Not needed but it does not hurt at all. It limits actions AaveDefensiveSeed contract can make

File: src/20240528_Multi_GHOCrossChainLaunch/AaveV3Arbitrum_GHOCrossChainLaunch_20240528.sol (hunk 2 above)
Comment (miguelmtzinf) [reply src/20240528_Multi_GHOCrossChainLaunch/AaveV3Arbitrum_GHOCrossChainLaunch_20240528.sol:243]: done!

**Pull request merged**